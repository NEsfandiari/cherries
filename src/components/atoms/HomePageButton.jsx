import React, { Component } from 'react'
import Styled from 'styled-components'
import { Link } from 'gatsby'

const Container = Styled.div`
background-color: #e20031;
color: white;
text-transform: uppercase;
text-align: center;
padding: 10px 30px 
font-size: 23px;
font-weight: 700;
width: 16rem;
cursor: pointer;
a{
  color: white;
  text-decoration: none;
}
`

class HomePageButton extends Component {
  render() {
    return (
      <Container>
        <Link to="/">GET STARTED</Link>
      </Container>
    )
  }
}

export default HomePageButton
