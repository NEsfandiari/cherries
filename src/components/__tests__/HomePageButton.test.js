import React from 'react'
import renderer from 'react-test-renderer'
import HomePageButton from '../atoms/HomePageButton'

describe('HomePageButton', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<HomePageButton />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
