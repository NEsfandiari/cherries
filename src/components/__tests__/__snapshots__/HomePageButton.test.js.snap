// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePageButton renders correctly 1`] = `
<div
  className="sc-bdVaJa kFEIDT"
>
  <a
    href="/subscribe"
  >
    GET STARTED
  </a>
</div>
`;
